tests/test_home_page.py

```diff
@@ -1,23 +1,44 @@
+'''
+Checks functionality of HomePage
+'''
 from utilities.base_test_case import BaseTestCase
 from pages.home_page import HomePage
-import time
 
 
-class TestOpenHomePage(BaseTestCase):
+class TestHomePage(BaseTestCase):
+    '''
+    Class for HomePage tests
+    '''
 
-    def test_OpenHomePage(self):
+    def test_open_home_page(self):
         '''
         Verifies if the HomePage opens
         '''
-        assert self.driver.title == "Google"
+        home_page_title = "Google"
+        self.assertEqual(self.driver.title, home_page_title, "HomePage Failed to load")
```

Comment:
What if it just needed a couple of more seconds to load? You created the wait_for_element_to_load that can be used here. 
Also, this assert checks the title and not that the page has loaded. What if the page loaded but with a different title? You should either change the way you test it or at least be more specific in your error message.

Reply:
Can I use is_displayed method to check if the element is present and based on that say that the page has loaded?